### PR TITLE
Add 'include_koji_repo' user parameter

### PR DESCRIPTION
**Pull request by owtaylor**
Add a user parameter 'force_tag_repo' with the meaning "include the
build tag repository from koji even yum_repourls" is set. This will
be used by the atomic reactor resolve_composes plugin to avoid
suppressing the build tag repository in circumstances where that
isn't desired.

Signed-off-by: Owen W. Taylor <otaylor@fishsoup.net>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [ ] Pull request includes link to an osbs-docs PR for user documentation updates

### CODE REVIEW

**Comment by twaugh** [comment]
The name seems a bit confusing to me: 'force tag' is a commonly used term. Maybe 'include_koji_tag'?

**Comment by owtaylor** [comment]
> The name seems a bit confusing to me: 'force tag' is a commonly used term. Maybe 'include_koji_tag'?

I have no strong attachment to the name - just need to shorten "force_include_koji_target_build_tag_repository" to some manageable length. :-) `include_koji_tag` is sounds like it has something to do with tagging the resulting builds. Maybe `include_koji_repo`?

**Comment by twaugh** [comment]
`include_koji_repo` sounds fine to me.

**Comment by owtaylor** [comment]
New version updates parameter name.

File: osbs/build/user_params.py [hunk 1]
```diff
@@ -416,6 +418,8 @@ def set_params(self,
         pkg_manager:name:version[:new_name]
         :param filesystem_koji_task_id: int, Koji Task that created the base filesystem
         :param flatpak: if we should build a Flatpak OCI Image
+        :param include_koji_repo: include the repo from the target build tag, even if other
```
Comment (mlangsdorf) [inline osbs/build/user_params.py]: Please put params in alphabetical order.

**Comment by owtaylor** [comment]
Re-alphabetized as necessary (got a little hasty with the search-and-replace when renaming :-)

**Comment by MartinBasti** [comment]
no release notes as this is only support for atomic-reactor patch (atomic reactor includes release notes)

**Pull request merged**